pkg/configs/updaters.go

```diff
@@ -0,0 +1,234 @@
+package configs
```

Comment:
Nice!  I wonder at some point it might be nice to have a test or two as we'll probably want to reuse this for future automation, so a test would make it easier to extend?  We've got some unit tests in this repo now so maybe we could reuse the approach?

Reply:
awesome sounds good to me!